src/components/rating/rating.md

```diff
@@ -0,0 +1,7 @@
+        const UiKit = require('../../');
```

Comment:
I'm in the process of upgrading styleguidist, and from now on code examples need to be wrapped in code blocks: 

\```js
\<Code />
\```

Just wanted to spread that knowledge 😃